sdk/samples/iot/provisioning/src/generate_certificate.sh

```diff
@@ -19,5 +23,10 @@ echo    export AZ_IOT_DEVICE_X509_TRUST_PEM_FILE=$(pwd)/BaltimoreCyberTrustRoot.
 
 echo -e "\nSample certificate generated:"
 echo "Upload device_ec_cert.pem to Device Provisioning Service."
-echo "Use the device_cert_store.pem file within the sample:"
+echo "Use the following command to set the environment variable for the samples:"
 echo    export AZ_IOT_DEVICE_X509_CERT_PEM_FILE=$(pwd)/device_cert_store.pem
+
+echo -e "\nUse the following fingerprint when enrolling your device in Device Provisioning Service:"
+openssl x509 -noout -fingerprint -in device_ec_cert.pem |  sed 's/://g'| tee  fingerprint.txt
+echo -e "\nThe fingerprint has also been placed in fingerprint.txt for future reference"
+notepad
```

Comment:
newline?

Reply:
It didn't have fingerprint in the right format before.  This makes the DPS sh version line up with the Hub sh version.  Main difference now is commenting changes.

Reply:
oh you're talking about notepad.  uh, yeah not sure what that is! lol. thanks.

Reply:
no I mean a newline at the end of the file haha.

Reply:
that too lolol. notepad was definitely bogus!